rotkehlchen/tests/db/test_reports.py

```diff
@@ -54,3 +70,122 @@ def test_report_settings(database):
         else:
             value = getattr(settings, setting_name)
         assert returned_settings[x] == value
+
+
+def test_report_events_sort_by_columns(database):
+    """Test that sorting by asset, pnl_taxable and timestamp works correctly"""
+    timestamp_1_secs = Timestamp(1741634066)
+    timestamp_2_secs = Timestamp(1741634100)
+    eth_price_ts_1 = FVal('2000')
+    eth_price_ts_2 = FVal('2200')
+    half_amount = FVal(0.5)
+    hundred = FVal('100')
+
+    dbreport, settings = setup_db_account_settings(database)
```

Comment:
For tests we can do this to save space

```suggestion
    timestamp_1_secs, timestamp_2_secs, eth_price_ts_1, eth_price_ts_2, half_amount, hundred  = Timestamp(1741634066), Timestamp(1741634100), FVal('2000'), FVal('2200'), FVal(0.5), FVal('100')  # noqa: E501
    dbreport, settings = setup_db_account_settings(database)
```